Lib/wsgiref/validate.py

```diff
@@ -137,7 +137,7 @@ def validator(application):
 
     """
     When applied between a WSGI server and a WSGI application, this
-    middleware will check for WSGI compliancy on a number of levels.
+    middleware will check for WSGI compliance on a number of levels.
```

Comment:
'compliancy' is an alternate spelling of 'compliance' which may or may not have a narrower or slightly different meaning, depending on the source, and which may be preferred in certain contexts or narrow communities.  Since it is possible that the author of the sentence consciously chose it, I am reverting.